solutions/standard/outputs.tf

```diff
@@ -47,11 +47,11 @@ output "keys" {
 }
 
 output "kp_private_endpoint" {
-  description = "Key Protect instance private endpoint URL when an instance is created, otherwise null"
+  description = "Key Protect instance private endpoint URL."
```

Comment:
It could be HPCS right? So we would have to rename this output to `kms_private_endpoint` and update the descrption

Reply:
I have changed kp -> kms and Key Protect -> Key Management Service in `solutions/standard/outputs.tf`, do we want to make same change in main module?